.storybook/config.js

```diff
@@ -0,0 +1,7 @@
+import { configure } from '@storybook/react';
+
+function loadStories() {
+    require('../stories/index.js');
```

Comment:
Super pedantic, but this is 4 spaces :)

Reply:
No such thing as being too pedantic when it comes to that kind of thing, in my opinion ;) Should set up Prettier to auto-format on commit, but for now I'll correct this, or else it'll drive me mad! Ta :)